src/theme/components/multiSelect.ts

```diff
@@ -8,7 +8,7 @@ export const multiSelectWidths = {
       md: "175px",
     },
     maxWidth: {
-      md: "200px",
+      md: "412px",
```

Comment:
From the other PR, I know this is to accommodate DXP's use case where library names are very long, but a more rounded number may fit better in different layouts. Something for @bigfishdesign13  to also take a look at.

Reply:
There should be independent max and min width values for the button and list.  Please refer to the `width` prop in the spec doc: https://docs.google.com/document/d/1HHgUy4wU2F-l7eB3PJpF1YUpHBqanmPUaU7te-uOCHM/edit#heading=h.j19d68lnasi3

The max width for the list was supposed to be `300px`, but I've now bumped that up to `360px`.

Reply:
I updated it with `"360px" and will test in scout